apps/etterlatte-api/src/main/kotlin/no/nav/etterlatte/samordning/sak/BehandlingSakRoutes.kt

```diff
@@ -59,7 +56,7 @@ fun Route.behandlingSakRoutes(
             }
         }
 
-        route("/${SAKID_CALL_PARAMETER}") {
+        route("/{$SAKID_CALL_PARAMETER}") {
```

Comment:
Obs obs

Reply:
Oi, bra catch. Ser det snek seg med i denne: 9db8521effcf2247d67ea8bfe3deaf6431d72d17

Her ser vi nok et eksempel på hvorfor 404 er problematisk å bruke for både "url finnes ikke" og "sak finnes ikke"... 🤷‍♂️